lib/node_modules/@stdlib/blas/ext/base/sapx/README.md

```diff
@@ -117,7 +113,7 @@ sapx.ndarray( 3, 5.0, x, 1, x.length-3 );
 
 ## Notes
 
--   If `N <= 0`, both functions return `x` unchanged.
+-   If `N <= 0`, both functions return the input array unchanged.
```

Comment:
Would you mind updating the example below, as well?

Reply:
I have updated the example to use utilities for generating the array